README.md

```diff
@@ -72,9 +72,11 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [DevScope](https://devscope.net/) [:rocket:](https://devscope.net/careers/) | Technology development, consultancy and product development. | `Porto` `Viseu` |
 | [Equal Experts](https://www.equalexperts.com) | Making software. Better. | `Remote` |
 | [Findmore](https://findmore.pt/) [:rocket:](https://findmore.pt/careers) | Nearshoring, software development, UX/UI Design and PMO. | `Lisboa` `Porto` <br> `Viseu` |
+| [Fyld](https://www.fyld.pt) [:rocket:](https://www.fyld.pt/it-jobs) | Consultancy with outsourcing services. Part of Joyn Group companies. | `Lisboa` |
 | [GLanDrive](https://www.glandrive.pt) | Software development and Consultancy. | `Lisboa` `Porto` |
-| [Growin](https://www.growin.com) [:rocket:](https://www.growin.com/it-jobs-growin-careers/) | Software development and system analysis consultancy. | `Lisboa` `Porto` |
+| [Growin](https://www.growin.com) [:rocket:](https://www.growin.com/it-jobs-growin-careers/) | Software development and system analysis consultancy. Part of Joyn Group companies. | `Lisboa` `Porto` |
```

Comment:
```suggestion
| [Growin](https://www.growin.com) [:rocket:](https://www.growin.com/it-jobs-growin-careers/) | Software development and system analysis consultancy. | `Lisboa` `Porto` |
```
I'm not sure this is relevant.

Reply:
Hey there. Do you want to remove `Part of Joyn Group companies.` as it is too much information?
I've applied it to the 3 companies, Growin, Fyld, and Landskill.

If you guys don't find it relevant, I have no problem removing it :)

Reply:
I'd say so, but let's wait on others' opinion

Reply:
Agree, "Part of  Joyn Group companies" seems like noise to me.

Reply:
@vpereira01 Hey there,

Just removed that info from the companies I added.

Please have a look.
Cheers,